docker/go-1.11.0/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Go cross compiler (xgo): Go 1.11
+# Copyright (c) 2018 Péter Szilágyi. All rights reserved.
+#
+# Released under the MIT license.
+
+FROM karalabe/xgo-base
+
+MAINTAINER Péter Szilágyi <peterke@gmail.com>
+
+# Configure the root Go distribution and bootstrap based on it
+ENV GO_VERSION 1100
```

Comment:
Should this be `GO_VERSION 1110`? Go 1.10.0 already uses `1100` https://github.com/karalabe/xgo/blob/master/docker/go-1.10.0/Dockerfile#L11

Reply:
good catch. Fixed.